pharo/run.sh

```diff
@@ -106,10 +106,13 @@ moose::get_image_url() {
       echo "https://ci.inria.fr/moose/job/${moose_name}/lastSuccessfulBuild/artifact/${moose_name}.zip"
       ;;
     "Moose64-8"*)
-      echo "https://github.com/moosetechnology/Moose/releases/download/v8.x.x/Moose8-stable.zip"
+      echo "https://github.com/moosetechnology/Moose/releases/download/v8.x.x/Moose8-old-stable-Pharo64-8.0.zip"
       ;;
     "Moose64-9"*)
-      echo "https://github.com/moosetechnology/Moose/releases/download/continuous/Moose9-development.zip"
+      echo "https://github.com/moosetechnology/Moose/releases/download/v9.x.x/Moose9-stable-Pharo64-9.0.zip"
+      ;;
+    "Moose64-10"*)
+      echo "https://github.com/moosetechnology/Moose/releases/download/continuous/Moose10-development-Pharo64-10.zip"
```

Comment:
Looking at the URL, the version scheme might indeed have changed. Could you please clarify?